slice/Ice/EndpointTypes.ice

```diff
@@ -10,38 +10,42 @@
 [["cpp:include:Ice/Config.h"]]
 [["cpp:include:cstdint"]]
 
+[["java:package:com.zeroc"]]
 [["js:module:@zeroc/ice"]]
-
 [["python:pkgdir:Ice"]]
 
-[["java:package:com.zeroc"]]
+// The endpoint types are called transport codes in IceRPC. They are used to marshal endpoints (as part of proxies) with
+// the Slice 1.x encoding.
 
 module Ice
 {
-    /// Uniquely identifies TCP endpoints.
+    /// Identifies endpoints marshaled as URI strings.
+    const short URIEndpointType = 0;
```

Comment:
Is this used anywhere, or just for completeness' sake?
More for curiosity than a 'review comment'.

Reply:
Thanks!